tests/unit/lib/config/Config.spec.js

```diff
@@ -1,4 +1,4 @@
-'use strict'; // eslint-disable-line
+'use strict';  
```

Comment:
extra spaces at end of line
```suggestion
'use strict';
```